### PR TITLE
Change configuration file format for terraform 0.12 and later

### PR DESCRIPTION
**Pull request by naoshark**
Change configuration file format for terraform 0.12 and later

### CODE REVIEW

File: website/docs/r/eri_port_to_azure_private_connection_v1.html.markdown [hunk 1]
```diff
@@ -16,34 +16,34 @@ Manages a V1 Port to Azure Private Connection resource within Flexible InterConn
...
-        end = 512
-    }
+  name = "terraform_port_1"
+  switch_name = "switch_name"
+  port_type = "1G"
+
+  vlan_ranges {
+    start = 497
+    end = 512
+  }
```
Comment (shuhei14) [inline website/docs/r/eri_port_to_azure_private_connection_v1.html.markdown]:
```suggestion
  name        = "terraform_port_1"
  switch_name = "switch_name"
  port_type   = "1G"

  vlan_ranges {
    start = 497
    end   = 512
  }
```

File: website/docs/r/eri_port_to_azure_private_connection_v1.html.markdown [hunk 2]
```diff
@@ -16,34 +16,34 @@ Manages a V1 Port to Azure Private Connection resource within Flexible InterConn
...
-    source_primary_port_id = "${fic_eri_port_v1.port_1.id}"
-    source_primary_vlan = 497
-    source_secondary_port_id = "${fic_eri_port_v1.port_1.id}"
-    source_secondary_vlan = 498
-    source_asn = "65530"
+  source_primary_port_id = fic_eri_port_v1.port_1.id
+  source_primary_vlan = 497
+  source_secondary_port_id = fic_eri_port_v1.port_1.id
+  source_secondary_vlan = 498
+  source_asn = "65530"
```
Comment (shuhei14) [inline website/docs/r/eri_port_to_azure_private_connection_v1.html.markdown]:
```suggestion
  source_primary_port_id   = fic_eri_port_v1.port_1.id
  source_primary_vlan      = 497
  source_secondary_port_id = fic_eri_port_v1.port_1.id
  source_secondary_vlan    = 498
  source_asn               = "65530"
```

File: website/docs/r/eri_port_to_azure_private_connection_v1.html.markdown [hunk 3]
```diff
@@ -16,34 +16,34 @@ Manages a V1 Port to Azure Private Connection resource within Flexible InterConn
...
 
-    destination_interconnect = "Osaka-1"
-    destination_qos_type = "guarantee"
-    destination_service_key = "service_key"
-    destination_shared_key = "shared_key"
+  destination_interconnect = "Osaka-1"
+  destination_qos_type = "guarantee"
+  destination_service_key = "service_key"
+  destination_shared_key = "shared_key"
```
Comment (shuhei14) [inline website/docs/r/eri_port_to_azure_private_connection_v1.html.markdown]:
```suggestion
  destination_interconnect = "Osaka-1"
  destination_qos_type     = "guarantee"
  destination_service_key  = "service_key"
  destination_shared_key   = "shared_key"
```

File: website/docs/r/eri_port_to_azure_private_connection_v1.html.markdown [hunk 4]
```diff
@@ -16,34 +16,34 @@ Manages a V1 Port to Azure Private Connection resource within Flexible InterConn
...
 
-    primary_connected_network_address = "10.10.0.0/30"
-    secondary_connected_network_address = "10.20.0.0/30"
+  primary_connected_network_address = "10.10.0.0/30"
+  secondary_connected_network_address = "10.20.0.0/30"
```
Comment (shuhei14) [inline website/docs/r/eri_port_to_azure_private_connection_v1.html.markdown]:
```suggestion
  primary_connected_network_address   = "10.10.0.0/30"
  secondary_connected_network_address = "10.20.0.0/30"
```

**Review by shuhei14 — APPROVED** [review]

File: website/docs/index.html.markdown [hunk 5]
```diff
@@ -18,11 +18,20 @@ Use the navigation to the left to read about the available resources.
 
 ```hcl
 # Configure the Flexible InterConnect Provider
+terraform {
+  required_providers {
+    fic = {
+      source  = "nttcom/fic"
+      version = "x.y.z"
```
Comment (ManabuMiwa) [inline website/docs/index.html.markdown]:
```suggestion
      version = "~> 0.5.3"
```
ECL と同じ主旨のコメントです。

**Review by ManabuMiwa — APPROVED** [review]

**Pull request merged**